### ISSUE TITLE
aggregate metrics send to monitoring

### ISSUE THREAD

**Issue by mehrdad2000** [resilience4j/resilience4j#2238]
Hi
how can i aggregate metrics that send to monitoring?
Currently resilience send lots zero value metrics in mili second to influxdb and it has used lots of disk size!

Any idea?
Thanks

**Comment by RobWin** [comment]
Hi,
what kind of InfluxDB integration are you using? 
Are you use Micrometer InfluxDB? 
Can you provide more information? What kind of zero value metrics?

**Comment by mehrdad2000** [comment]
@RobWin Yes Micrometer InfluxDB, creates lots of metrics like this in microsecond time units!
Looking for a way to aggregate them or any way to reduce data points.
Here is example, it could be better if it only send close 1 to influx instead send other zero items!

resilience4j_circuitbreaker_state{name="backendA",state="half_open",} 0.0
resilience4j_circuitbreaker_state{name="backendA",state="forced_open",} 0.0
resilience4j_circuitbreaker_state{name="backendA",state="disabled",} 0.0
resilience4j_circuitbreaker_state{name="backendA",state="closed",} 1.0
resilience4j_circuitbreaker_state{name="backendA",state="open",} 0.0
resilience4j_circuitbreaker_state{name="backendA",} 0.0


Any idea?
